lib/touchbar.es

```diff
@@ -0,0 +1,110 @@
+const {app, BrowserWindow, TouchBar} = require('electron')
```

Comment:
Wondering why. Simmilar <code>const</code> <code>require</code> syntax is also used by app.js, like this:
<pre>
const {app, BrowserWindow, ipcMain, Tray, nativeImage, shell} = require('electron')
</pre>